pkg/kubernetes/smihelper/errors.go

```diff
@@ -0,0 +1,16 @@
+package smihelper
+
+import (
+	"errors"
+)
+
+var (
+	// ErrGettingServices is used when a services for a given pod's namespace cannot be found
+	ErrGettingServices = errors.New("services for pod's namespace could not be found")
```

Comment:
Generally I've seen errors for objects not found be called something like `ErrServicesNotFound` or for this it might be ErrServicesNotFoundForPod. That kind of var name I think more appropriately reflects the description above. What do you think?

Reply:
Yep that makes sense